api/data_test.go

```diff
@@ -149,3 +149,175 @@ func TestCreateNewCacheTime(t *testing.T) {
 		})
 	})
 }
+func TestCreateOrUpdateCacheTimeReturnsErr(t *testing.T) {
+	const validBody = `{"path": "testpath", "etag": "testetag"}`
+
+	Convey("Given a mock API", t, func() {
```

Comment:
Small one, but I think the API we're testing here is not a mock, it's a real one that uses a mock data store. So you can just change this to something like "Given an API", or similar.